README.rst

```diff
@@ -242,7 +242,7 @@ MXNet SageMaker Estimators
 
 By using MXNet SageMaker ``Estimators``, you can train and host MXNet models on Amazon SageMaker.
 
-Supported versions of MXNet: ``1.2.1``, ``1.1.0``, ``1.0.0``, ``0.12.1``.
+Supported versions of MXNet: ``1.2.1``, ``1.1.0``, ``1.0.0``, ``0.12.1``. We recommend to use the latest version and our integration test will only test the latest version.
```

Comment:
I would add the sentence in a new line after the list of versions and change it to something like "We recommend you use the latest version as that's where we spend most of our development efforts". Maybe @djarpin or @eslesar-aws has a better wording suggestion.

Reply:
"We recommend that you use the latest supported version, because that's where we focus most of our development efforts."

Reply:
Thanks!